app/src/main/res/values/strings.xml

```diff
@@ -1,8 +1,26 @@
 <resources>
   <string name="app_name">Blood Donation</string>
+  <string name="log_in_label">Log in</string>
+  <string name="need_account">Need an account</string>
+  <string name="sing_up_text">Sign up now</string>
+  <string name="or_text">or</string>
+  <string name="email_or_phone_number">Email/Phone number</string>
+  <string name="password">Password</string>
+  <string name="login_with_facebook">Login with Facebook</string>
```

Comment:
@ritesh-singh we're not doing social intergrations right?

Reply:
Nope, I have removed and pushed.

Reply:
@intkhabahmed Remove extra strings.